apps/edxapp/vars/all/main.yml

```diff
@@ -34,3 +34,12 @@ edxapp_nginx_image_name: "fundocker/openshift-nginx"
 edxapp_nginx_image_tag: "1.13"
 edxapp_nginx_cms_port: 8081
 edxapp_nginx_lms_port: 8071
+
+# -- celery/redis
+edxapp_celery_lms_high_priority_queue: "edx.lms.core.high"
+edxapp_celery_lms_default_priority_queue: "edx.lms.core.default"
+edxapp_celery_lms_low_priority_queue: "edx.lms.core.low"
+edxapp_celery_lms_high_mem_queue: "edx.lms.core.high_mem"
+edxapp_celery_cms_high_priority_queue: "edx.cms.core.high"
```

Comment:
maybe split in two groups lms and cms?

Reply:
What do you mean by "group"?

Reply:
add a comment line to split in two groups... just visual